todoist-card.js

```diff
@@ -505,10 +595,9 @@ class TodoistCard extends LitElement {
                                         icon="mdi:circle-medium"
                                     ></ha-icon>`}
                                 <div class="todoist-item-text">
-                                    ${item.description
-                                        ? html`<span class="todoist-item-content">${item.content}</span>
-                                            <span class="todoist-item-description">${item.description}</span>`
-                                        : item.content}
+                                    ${item.content ? html`<span class="todoist-item-content">${item.content}</span>` : null}
+                                    ${item.description ? html`<span class="todoist-item-description">${item.description}</span>` : null}
+                                    ${item.due ? html`<span class="todoist-item-due">${item.due.date}</span>` : null}

```

Comment:
Won't there be a problem with margins if `item`'s `description` and `due` are empty?

Reply:
Using `?` as a ternary operator is safe to use to check if those are null and to render null html if so, otherwise render the corresponding data with it. This allows the card to have all 3 of these properties as optionals.

The css could probably use a little work as I'm very amateur with front-end code so it's a little rough:

![image](https://user-images.githubusercontent.com/87995947/214914924-ac82b0ba-8d60-4e67-972f-e84b5239fda0.png)

Reply:
What I'm trying to say is that if `description` and `due` fields are empty, `content` will be rendered in a `<div>` with `todoist-item-content` class, which has top and bottom margins.

Reply:
Yeah I can give it a shot sometime this week - I should have more time this weekend

Reply:
Seems like it's behaving about the same as the other code I think.

Your code:
![image](https://user-images.githubusercontent.com/87995947/215403101-1acc3c24-8edd-4899-8085-eec3eb883446.png)

My previous code:
![image](https://user-images.githubusercontent.com/87995947/215403560-b345419a-f04a-4d9b-9d01-5c1ba853c65d.png)

Reply:
I think it's just cache. In my code, `description` and `due.date` and separated by a vertical bar.

Reply:
Ah yeah I see that now, I'll try again soon(ish)

Reply:
@sw-carlos-cristobal did you have some time to take a look at this?